pkg/client/unversioned/helper.go

```diff
@@ -672,3 +709,18 @@ func TimeoutFromListOptions(options api.ListOptions) time.Duration {
 	}
 	return 0
 }
+
+func stripAPIFromPrefix(base string) string {
+	// normalize the base
```

Comment:
The case apis/ does not happen anymore? Was working around that before in absPath().